scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,218 @@
+
+    tools_build_path = eutil.handle_tools(ctx, should_build_tools, is_installer_sdk, tool_config, '', tools_build_path, default_base_path, engine_centric, logger)
+
+    eutil.handle_assets(ctx, should_build_all_assets, tools_build_path, is_installer_sdk, 
+                        tool_config, engine_centric, fail_on_asset_errors,
+                        ASSET_PLATFORM, ASSET_PLATFORM, validated_seedslist_paths, preprocessed_seedfile_paths, 
+                        ctx.project_path/'AssetBundling', max_bundle_size,
+                        logger)
+
+    android_project_config= android_support.get_android_config(project_path=None)
+
+    android_sdk_home = pathlib.Path(android_project_config.get_value(key=android_support.SETTINGS_SDK_ROOT.key))
+
+    run_android_command([o3de_cli_script_path, 'android-generate', '-p', ctx.project_name, '-B', target_android_project_path, "--asset-mode", asset_pack_mode],
+                        cwd=default_base_path,
+                        error_msg="Android Project Generation Failure.")
+
+    run_android_command([target_android_project_path / ('gradlew'+ android_support.GRADLE_EXTENSION), f'assemble{build_config.title()}'],
+                        cwd=target_android_project_path,
+                        error_msg="Gradle Build Failure.")
+    
+    if deploy_to_device:
+        if not activity_name:
+            activity_name = f'{ctx.project_name}Activity'
+
+        if not org_name:
+            org_name = f'org.o3de.{ctx.project_name}'
+        android_support.deloy_to_android_device(target_android_project_path,
+                                build_config,
+                                android_sdk_home,
+                                org_name,
+                                activity_name)
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext, export_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as standalone to the desired output directory with release layout. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    
+    eutil.create_common_export_params_and_config(parser, export_config)
+
+    parser.add_argument('-deploy', '--deploy-to-android',default=False,action='store_true',help='At completion of build, deploy to android device.')
+
+    parser.add_argument('-actname', '--activity-name', type=str, default=f'{o3de_context.project_name}Activity',
+                        help='The name of the activity to invoke when deploying and running the APK.')
+    
+    parser.add_argument('-orgname', '--org-name', type=str, default=f'org.o3de.{o3de_context.project_name}',
```

Comment:
No need for a short-form of the argument, especially when its about the same length as the long form
```suggestion
    parser.add_argument('--org-name', type=str, default=f'org.o3de.{o3de_context.project_name}',
```